README.md

```diff
@@ -76,6 +80,19 @@ This returns the the page's `content` as GitHub-flavored Markdown:
 "content": "...**Thunder** is the [stage name](https://en.wikipedia.org/wiki/Stage_name) for the..."
 ```
 
+##### Custom HTML
```

Comment:
Let's make this `Pre-fetched HTML`. It's a lot more often going to be the use case vs. "custom" html, which is a little more ambiguous.